lib/match.js

```diff
@@ -20,7 +20,7 @@ util.inherits(MatchStream, Readable)
 MatchStream.prototype.scan = function () {
   var self = this
 
-  this.opt.redis.eval(lua, 0, this.opt.match, function (err, res) {
+  this.opt.redis.eval(lua, 0, this.opt.match, function evalLua (err, res) {
```

Comment:
Alright. Also, there's a try-catch in there for the JSON.parse. Would a regex check be optimal instead ?